mods/ra/rules/structures.yaml

```diff
@@ -1116,6 +1116,7 @@ HPAD:
 		ZOffset: 256
 		UpgradeTypes: primary
 		UpgradeMinEnabledLevel: 1
+	WithRearmAnimation:
```

Comment:
They definitely do artwork-wise, but they will be overlays instead of animations, and therefore never hooked up.  Another PR can add `WithRearmOverlay` to fix that.